cmake/Modules/FindCatch_EP.cmake

```diff
@@ -63,10 +63,10 @@ message(VERBOSE "CATCH_INCLUDE_DIR is \"${CATCH_INCLUDE_DIR}\", CATCH_LIBRARIES
 if(Catch2_FOUND)
   message(VERBOSE "Catch2_FOUND is ${Catch2_FOUND}, CATCH_INCLUDE_DIR is \"${CATCH_INCLUDE_DIR}\", CATCH_LIBRARIES is \"${CATCH_LIBRARIES}\"")
 else()
-  message(VERBOSE "TILEDB_SUPERBUILD is ${TILEDB_SUPERBUILD}, Catch2_FOUND is ${Catch2_FOUND}, CATCH2_FOUND is ${CATCH2_FOUND}")
+  message(VERBOSE "TILEDB_SUPERBUILD is ${TILEDB_SUPERBUILD}, Catch2_FOUND is ${Catch2_FOUND}, Catch2_FOUND is ${Catch2_FOUND}")
```

Comment:
The last three terms repeat the preceding three.

Reply:
Did you catch this before I pushed a fix that looked for all of the `CATCH2_FOUND` instances? There were exactly three. Two on this line and one towards the top.

Reply:
@eddelbuettel means that `Catch2_FOUND is ${Catch2_FOUND}` is repeated verbatim.

Reply:
Oh duh. Gotchya.

Reply:
Fixed.